template-compose/app/build.gradle.kts

```diff
@@ -142,16 +144,13 @@ dependencies {
     releaseImplementation("com.github.chuckerteam.chucker:library-no-op:${Versions.CHUCKER_VERSION}")
 
     // Testing
+    testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:${Versions.KOTLINX_COROUTINES_VERSION}")
     testImplementation("io.kotest:kotest-assertions-core:${Versions.TEST_KOTEST_VERSION}")
     testImplementation("junit:junit:${Versions.TEST_JUNIT_VERSION}")
-    testImplementation("androidx.test:core:${Versions.TEST_ANDROIDX_CORE_VERSION}")
-    testImplementation("androidx.test:runner:${Versions.TEST_RUNNER_VERSION}")
-    testImplementation("androidx.test:rules:${Versions.TEST_RUNNER_VERSION}")
-    testImplementation("androidx.test.ext:junit-ktx:${Versions.TEST_JUNIT_ANDROIDX_EXT_VERSION}")
-    testImplementation("com.google.dagger:hilt-android-testing:${Versions.HILT_VERSION}")
-    testImplementation("org.jetbrains.kotlin:kotlin-reflect:${Versions.KOTLIN_REFLECT_VERSION}")
     testImplementation("io.mockk:mockk:${Versions.TEST_MOCKK_VERSION}")
+    testImplementation("app.cash.turbine:turbine:${Versions.TEST_TURBINE}")
 
-    kaptTest("com.google.dagger:hilt-android-compiler:${Versions.HILT_VERSION}")
-    testAnnotationProcessor("com.google.dagger:hilt-android-compiler:${Versions.HILT_VERSION}")
+    // Need to have BOM for androidTestImplementation https://github.com/gradle/gradle/issues/23347
+    androidTestImplementation(platform("androidx.compose:compose-bom:${Versions.COMPOSE_BOM_VERSION}"))
+    androidTestImplementation("androidx.compose.ui:ui-test-junit4")
```

Comment:
@Wadeewee Yes, I think we should recommend running UI tests with Robolectric by default. In case we need to run them with the Instrument test, we can look at this sample later.

FYI, I will update this PR to set up running UI tests with Robolectric by default https://github.com/nimblehq/android-templates/pull/399 cc @Tuubz @Wadeewee